.github/workflows/review-feedback-loop.yml

```
@@ -58,6 +58,23 @@ jobs:
+    # prices at roughly 100 points. The 5,000 points/hour budget is therefore about
+    # 50 sweeps, and an agent pushing a few times across a few branches exhausts it;
+    # `sweep-review-threads` then fails on every PR in the repo with
+    # "API rate limit already exceeded".
+    #
+    # Cancelling a superseded sweep is safe *because sync-pr is a sync*: it drives
+    # labels and thread state toward whatever the PR currently looks like. A run
+    # interrupted halfway is corrected by the next run, which reads current state
+    # rather than resuming. The only run whose verdict matters is the last one.
+    #
+    # Job-level, not workflow-level, so the issue_comment jobs are untouched.
+    concurrency:
```

actionlint (1.7.12)

[error] key "concurrency" is duplicated in "sweep-review-threads" job. previously defined at line:57,col:5

(syntax-check)